src/components/Board/Board.tsx

```diff
@@ -0,0 +1,12 @@
+import { HomeBody } from "components/HomeBody";
+import Loading from "components/Loading/Loading";
+import { useSelector } from "react-redux";
+import { ClipLoader } from "react-spinners";
+
+const Board = () => {
+  const isLoading = useSelector((state: any) => state.isLoading);
+
+  return <>{!isLoading ? <Loading /> : <HomeBody />}</>;
+};
```

Comment:
just change HomeBody component to Board component and check loading there. And do not need fragment